### ISSUE TITLE
Colorings as array in Auspice config JSON

### ISSUE THREAD

**Issue by trvrb** [nextstrain/augur#401]
@jameshadfield @emmahodcroft ---

This issue was originally raised in #344 and originally addressed in PR #361. In these the exported `colorings` were moved from a dict to an array. This is correctly exported by `augur export v2` and correctly ingested by auspice v2. However, the Auspice config JSON was never updated to move from a dict to an array.

This can be seen in the Zika test build: https://github.com/nextstrain/augur/blob/v6/tests/builds/zika/config/auspice_config_v2.json#L3. If I modify this to be:
```
  "colorings": [
    {
      "key": "gt",
      "title": "Genotype",
      "type": "categorical"
    },
    {
      "key": "num_date",
      "title": "Sampling date",
      "type": "continuous"
    },
    {
      "key": "author",
      "title": "Authors",
      "type": "categorical"
    },
    {
      "key": "country",
      "type": "ordinal"
    },
    {
      "key": "region",
      "type": "ordinal"
    }
  ],
```
I get an error on export `AttributeError: 'list' object has no attribute 'keys'`.

Without this fix, it wouldn't be possible to specify colorings order in the Auspice config.

**Comment by jameshadfield** [comment]
👍 It seems sensible to have the config file mirror the produced JSONs as much as possible.

This would require:
* changes to the config schema: https://github.com/nextstrain/augur/blob/v6/augur/data/schema-auspice-config-v2.json#L12
* changes to the export v2 code
* changes to all core builds which use a config file -- see #334

> Without this fix, it wouldn't be possible to specify colorings order in the Auspice config.

JSON object key ordering is maintained by the parsers which augur uses. But I agree this change would be good to implement now.

**Comment by trvrb** [comment]
Resolved in #406
